src/js/components/DataChart/Detail.js

```diff
@@ -172,6 +180,7 @@ const Detail = ({
                 onMouseOver={(event) => {
                   activeIndex.current = event.currentTarget;
                   setDetailIndex(i);
+                  announce(getContent(i), 'assertive');
```

Comment:
Totally blind users won't use a mouse (or any other visual pointing device) but I've left the previous mouse hover behaviour in, in case it's useful to announce the detail content to sighted VoiceOver users.